engine/table/src/main/java/io/deephaven/engine/table/impl/SliceLikeOperation.java

```diff
@@ -134,6 +145,10 @@ public void onUpdate(TableUpdate upstream) {
     }
 
     private void onUpdate(final TableUpdate upstream) {
+        if (parent.isBlink()) {
```

Comment:
We need to come up with a strategy to avoid creating this listener. We *could* create some kind of "no-op" listener. That said, it might be better to short-circuit up in `QueryTable` and never hit `SliceLikeOperation`.